assets/src/application/ui/display/icons/collection/Close.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import { IconProps, IconName } from '../types';
+import Icon from '../Icon';
+
+const Close = React.forwardRef<SVGSVGElement, Partial<IconProps>>((props, ref) => (
+	<Icon {...props} name={IconName.CLOSE} ref={ref} viewBox='0 0 24 24' />
```

Comment:
Any specific reason for passing `viewBox`?

Reply:
this how it was showing w/o that 

![image](https://user-images.githubusercontent.com/1477453/79443361-6742f800-7fe2-11ea-904d-40a3f30fc29e.png)

Reply:
w/ viewbox:

![image](https://user-images.githubusercontent.com/1477453/79443607-b4bf6500-7fe2-11ea-89a5-ea9b3224173d.png)

Reply:
OK, cool :+1: